### PR TITLE
View pokemon

### PR DESCRIPTION
**Pull request by KennethDelaCruz**

### CODE REVIEW

**Comment by KennethDelaCruz** [comment]
ajax-project

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey dude! Yur input and button are off-center when viewed on mobile, please revise 👍

**Comment by KennethDelaCruz** [comment]
ajax-project

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+searchForm.addEventListener('submit', submitForm);
+
+function test1(object){
+  var abilities = object.currentPokemon.abilities;
+  console.log(abilities)
+  for (var i = 0; i < abilities.length; i++) {
+    console.log(abilities[i]["ability"].name)
+  }
+
+}
```
Comment (codingcodymiller) [inline js/main.js]: Is this being used for anything?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**